app/lib/actions/katello/content_view_version/create_export_history.rb

```diff
@@ -0,0 +1,65 @@
+module Actions
+  module Katello
+    module ContentViewVersion
+      class CreateExportHistory < Actions::EntryAction
```

Comment:
Considering this file is doing a lot of things tied to pulp3 I think it needs to be in the same directory as the export/import for pulp3 -> `app/lib/actions/pulp3/orchestration/content_view_version`

Reply:
you're right, moving it